packages/x-components/src/x-modules/empathize/components/__tests__/empathize.spec.ts

```diff
@@ -1,53 +1,110 @@
-import { mount } from '@vue/test-utils';
+import { mount, Wrapper } from '@vue/test-utils';
 import Vue from 'vue';
-import BaseEventButton from '../../../../components/base-event-button.vue';
 import { getXComponentXModuleName, isXComponent } from '../../../../components/x-component.utils';
 import { getDataTestSelector, installNewXPlugin } from '../../../../__tests__/utils';
 import Empathize from '../empathize.vue';
+import { empathizeXModule } from '../../x-module';
+import { XPlugin } from '../../../../plugins/index';
```

Comment:
Avoid importing from barrel files

```suggestion
import { XPlugin } from '../../../../plugins/x-plugin';
```